app/auth_manager.py

```diff
@@ -120,30 +226,32 @@ async def get_user_manager(
 
 
 cookie_transport = CookieTransport(
-    cookie_max_age=TOKEN_EXPIRE, cookie_secure=SECURE_COOKIE
+    cookie_name="access_token",
+    cookie_max_age=ACCESS_TOKEN_EXPIRE,
+    cookie_secure=SECURE_COOKIE,
 )
 
 
-def get_jwt_strategy() -> JWTStrategy:
+def get_strategy() -> CustomJWTStrategy:
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (docstrings):** Please update the docstring for function: `get_strategy`

Reason for update: Function signature and purpose changed from JWTStrategy to CustomJWTStrategy with additional configuration parameters.

Suggested new docstring:
```
    """
    Returns a custom JWT strategy with specified secret and token lifetimes.

    This function creates and returns a CustomJWTStrategy object, which is configured with
    access and refresh token secrets, their lifetimes, and additional settings for secure
    token handling and validation.

    Returns:
        CustomJWTStrategy: The custom JWT strategy object.
    """
```